source/draw/gpu/opengl/YuvOptimizedContext.ooc

```diff
@@ -64,6 +65,10 @@ YuvOptimizedContext: class extends OpenGLContext {
 			result = this _monochromeToYuv
 		else if (output instanceOf(EGLYuv) && input instanceOf(GpuYuv420Semiplanar))
 			result = this _compositeYuvToNativeYuv
+		else if (input instanceOf(EGLYuv) && output instanceOf(OpenGLMonochrome))
```

Comment:
If you feel like it you could place all ```input instanceOf(EGLYuv)``` after one another.

Is there any need for support of render to RasterMonochrome? (I guess we simply add when needed)